spec.html

```diff
@@ -3766,7 +3777,7 @@ <h1>The Reference Record Specification Type</h1>
             </tr>
             <tr>
               <td oldids="sec-getreferencedname,ao-getreferencedname">[[ReferencedName]]</td>
-              <td>String or Symbol</td>
+              <td>String, Symbol or Private Name</td>
```

Comment:
:ahem:
```suggestion
              <td>String, Symbol, or Private Name</td>
```

Reply:
(That's already been [suggested](https://github.com/tc39/ecma262/pull/1668/files#r560402082) in PR #1668.)

Reply:
Done in 53322a0, and added to #1668.